gems/pending/appliance_console/database_replication.rb

```diff
@@ -0,0 +1,115 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'English'
+
+module ApplianceConsole
+  class DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REPMGR_CONFIG = '/etc/repmgr.conf'.freeze
+
+    attr_accessor :cluster_name, :node_number,
+                  :database_name, :database_user, :database_password,
+                  :primary_host, :standby_host
+
+    def ask_for_cluster_name
```

Comment:
I thought, we were going to query the database for this? Not sure why my previous comment was clobbered.